src/main/java/uk/gov/hmcts/reform/sscs/service/RegionalProcessingCenterService.java

```diff
@@ -98,21 +98,11 @@ public RegionalProcessingCenter getByScReferenceCode(String referenceNumber) {
     /**
      * Lookup by the name of the RPC. We should be getting rid of the above code to
      * get it from the SC Reference. Restructure the json file to work by just the name.
-     * @param name the RPC name
-     * @return RPC name
+     * @param name the RPC name e.g. "Birmingham"
+     * @return RPC object
      */
     public RegionalProcessingCenter getByName(String name) {
-        if (StringUtils.isBlank(name)) {
-            return null;
-        }
-
-        String regionalProcessingCenter = sccodeRegionalProcessingCentermap.get("SSCS " + name);
```

Comment:
This could **never** work because the sccodeRPCmap looks like this:
```
"SC112": "SSCS Venue0",
"SC358": "SSCS Venue1",
"SC132": "SSCS Venue2",
"SC134", "SSCS Venue3"
```